### ISSUE TITLE
Rune Chopper latent does not work

### ISSUE THREAD

**Issue by Falaras** [EdenServer/community#2032]
### Checklist

- [x] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [x] I have provided reproducable steps. (No "as the title says" posts please. Provide reproducable steps even if it seems like a waste of time.)
- [x] I have provided my client version in the details. (type /ver into your game window)

Rune Chopper latent is not currently being applied here. The mana drain is active but the Haste+9% is not, not sure on the Accuracy+5. Tested with Utsusemi: Ni recast was the same with no weapon, Rune Chopper draining MP, and Rune Chopper without MP left.

Ver 30181205_0

**Comment by Tjonke** [comment]
I can confirm this: 3 min /recast on Dread Spikes with and without the Rune Chopper equiped, and engaged.

**Comment by greyenvy** [comment]
Can /checkparam yourself to see of the accuracy is working or not.

**Comment by roriffxi** [comment]
using /checkparam the accuracy works for me.

Using stoneskin my recast was 24 sec without and 23 sec with.

**Comment by Falaras** [comment]
I decided to test it a bit more with gearswap turned off this time and using checkparam, the Acc is working as intended.

(0% Haste) recast time: 44 Seconds.

With Turban+Dusk (8% haste) recast time: 40 seconds.

Rune Chopper active draining MP (9% Haste) recast time: 43 seconds.

Rune Chopper drawn but OOM (0% haste) recast time: 44 seconds.

I also noticed this behavior when equipping more haste gear, I was generally lowering Utsusemi by barely 1 second with RC drawn. It should be closer to 4-5 seconds lowered when compared to other setups of Haste gear. Honestly not sure what's going on.
